src/sage/rings/number_field/number_field_element.pyx

```diff
@@ -3965,36 +3969,35 @@ cdef class NumberFieldElement(NumberFieldElement_base):
 
     def local_height_arch(self, i, prec=None, weighted=False):
         r"""
-        Returns the local height of self at the `i`'th infinite place.
+        Returns the local height of ``self`` at the `i`'th infinite place.
 
```

Comment:
Look https://deploy-preview-35457--sagemath-tobias.netlify.app/reference/number_fields/sage/rings/number_field/number_field_element.html#sage.rings.number_field.number_field_element.NumberFieldElement.local_height_arch

I don't know why. Perhaps it should be `i-th`.

Reply:
I meant the INPUT field.

Reply:
I also prefer `` `i`-th `` over other variants

Reply:
The bad indentation in the first INPUT is due to bad indentation. I'll fix it.